metastore/scripts/upgrade/hive/hive-schema-3.0.0.hive.sql

```diff
@@ -0,0 +1,1334 @@
+-- HIVE system db
+
+CREATE DATABASE SYS;
```

Comment:
I can definitely create a follow up, but I'm not sure this is such a great idea. The point of having these meta schemas is that you have a place to look up info programmatically. This will make it more complicated.